scripts/backports.py

```diff
@@ -0,0 +1,112 @@
+#!/bin/python
+
+# Looks for backports since a previous patch release.
+#
+# Requires:
+# - a github oauth token with public_repo and read:org scopes for kiali
```

Comment:
I'm an idiot :) I didn't read the comments. This is what I was missing.

Reply:
d'oh!